torch_geometric/explain/utils/metrics.py

```diff
@@ -0,0 +1,83 @@
+import torch
+from torch import Tensor
+
+
+def get_characterization_score(fidelity: Tensor,
+                               weights: Tensor = torch.tensor([1, 1])) \
+        -> 'Tensor':
+    r"""Returns the componentwise characterization score where fidelity[0]
+    is the tensor containing F_{+} and fidelity[1] is the tensor
+    containing F_{-}.
+
+    The characterization score is calculated as
+     .. math::
+      charac = \frac{w_+ + w_-}{\frac{w_+}{fid_+}+\frac{w_-}{fid_-}}
```

Comment:
I think there might be a typo in the formula here
```suggestion
      charac = \frac{w_+ + w_-}{\frac{w_+}{fid_+}+\frac{w_-}{1-fid_-}}
```